.github/workflows/ci_tests.yml

```diff
@@ -41,16 +41,15 @@ jobs:
         run:
           sudo apt install -y protobuf-compiler libssl-dev libpq-dev libsqlite3-dev pkg-config
 
-      - name: Run tests
+      - name: Run node tests
         uses: actions-rs/toolchain@v1
         with:
           toolchain: "1.65"  # it says it can read the rust-toolchain file, but it fails if we omit this
           components: rustfmt, clippy
       - run: |
-          cargo build -p jormungandr
-          cargo build -p jcli
-          cargo nextest run --no-fail-fast --partition hash:${{ matrix.shardNumber }}/${{ matrix.totalShards }}
-
+          cd src/jormungandr
+          cargo build
+          cargo nextest run -p jormungandr -p jcli -p jormungandr-integration-tests --no-fail-fast --partition hash:${{ matrix.shardNumber }}/${{ matrix.totalShards }}
```

Comment:
This means we're not testing `vit-servicing-station` right? Or am I missing something

Reply:
well I need to confess that this is experimental change to narrow down the issues with insufficient space. I need to add other tests in different jobs